File: telerik_ui_for_winforms_courseware_chm/Courseware/Grid/CS/RadGridView/RadGridViewIntro/RadGridViewLab1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace _01_RadGridViewIntro
{
    public partial class RadGridViewLab1 : Form
    {
        public RadGridViewLab1()
        {
            InitializeComponent();
        }

        private void RadGridViewLab1_Load(object sender, EventArgs e)
        {
            // TODO: This line of code loads data into the 'adventureWorksDataSet.SalesPerson' table. You can move, or remove it, as needed.
            this.salesPersonTableAdapter.Fill(this.adventureWorksDataSet.SalesPerson);

        }
    }
}

File: telerik_ui_for_winforms_courseware_chm/Courseware/Grid/VB/RadGridView/RadGridViewIntro/RadGridViewLab1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace _01_RadGridViewIntro
	Public Partial Class RadGridViewLab1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub RadGridViewLab1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' TODO: This line of code loads data into the 'adventureWorksDataSet.SalesPerson' table. You can move, or remove it, as needed.
			Me.salesPersonTableAdapter.Fill(Me.adventureWorksDataSet.SalesPerson)

		End Sub
	End Class
End Namespace
